boot/modeenv.go

```diff
@@ -59,9 +59,18 @@ type Modeenv struct {
 	TryBase             string   `key:"try_base"`
 	BaseStatus          string   `key:"base_status"`
 	CurrentKernels      []string `key:"current_kernels"`
-	Model               string   `key:"model"`
-	BrandID             string   `key:"model,secondary"`
-	Grade               string   `key:"grade"`
+	// Model, BrandID, Grade, SignKeyID describe the properties of current
+	// device model.
+	Model     string `key:"model"`
+	BrandID   string `key:"model,secondary"`
+	Grade     string `key:"grade"`
+	SignKeyID string `key:"sign_key_id"`
```

Comment:
we should probably follow the naming we used in the boot chains and call this:

ModelSignKeyID  model_sign_key_id